src/WCSimConstructMultiPMT.cc

```diff
@@ -288,7 +288,8 @@ G4LogicalVolume* WCSimDetectorConstruction::ConstructMultiPMT(G4String PMTName,
   // origin is center of upward cylinder
   G4LogicalVolume *logic_mPMT_container;
   //necessary to make concentric shells because can Mother can only contain parametrized daughters.
-  G4double innerR_curv_container = vessel_radius_curv - mPMT_outer_material_d - 54.*mm;  //-expose - dist_pmt_vessel
+  //G4double innerR_curv_container = vessel_radius_curv - mPMT_outer_material_d - 54.*mm;  //-expose - dist_pmt_vessel
+  G4double innerR_curv_container = vessel_radius_curv - mPMT_outer_material_d - 59.62*mm;  //updated number with support structure
```

Comment:
This new number comes from [here](https://github.com/WCTE/WCSim/blob/288204e52e9654aad377887418797c2b50b8cc19/src/WCSimConstructPMT.cc#L84)

Reply:
will this hardcoding in two different places be a problem if someone wants to change it and forgets one place?

Reply:
changed to directly use `pmtModuleHeight`